docs/contributing/pull-requests/chromatic.md

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_custom_props:
+  access: bitwarden
+---
+
+# UI Review - Chromatic
+
+We use a tool called [Chromatic][chromatic] to run automated snapshot tests for all Storybook
+stories. This allows us to quickly catch design regressions in an automated fashion. As part of this
+we use Chromatic to also review and approve the visual changes. Members of the Bitwarden GitHub
+organization can login to Chromatic using their GitHub account.
+
+Chromatic splits the review process into two parts, _UI Tests_ and _UI Review_.
+
+## UI Tests
```

Comment:
Could we put the UI Tests and UI Review sections as third-level titles (`###`) so that it's clear that they are sub-sections of the previous verbiage?

Reply:
We could, but we would need a title for that section. Without that the headers will just have a slightly smaller font size without actually being nested under anything.